wadl/solver/solver.py

```diff
@@ -85,7 +86,12 @@ def setup(self, graph):
     def solve(self, routeSet):
         startTime = time.time()
         # solve each tile
+        # save subPaths, metaGraph after initially found so that they can be reused
         subPaths = self.solveTiles()
+        self.subPathsCopy = copy.deepcopy(subPaths)
+        self.metaGraphCopy = copy.deepcopy(self.metaGraph)
+        # copy before merging tiles
+        self.copy = copy.deepcopy(self)
```

Comment:
why copy the whole object into itself?

Reply:
I copied the whole solver object since it is being changed in mergeTiles. Or do you suggest just copying the metagraph?

Reply:
each solver has a list of subpaths that won't change when you try to relink. Each solver also only has one metagraph so there shouldn't be any need for copying and caching the individual parts if we just keep a bunch of separate solver objects.

Reply:
got it, thanks!